app/app.go

```diff
@@ -98,12 +98,12 @@ import (
 	"github.com/medibloc/panacea-core/v2/x/burn"
 	burnkeeper "github.com/medibloc/panacea-core/v2/x/burn/keeper"
 	burntypes "github.com/medibloc/panacea-core/v2/x/burn/types"
+	"github.com/medibloc/panacea-core/v2/x/datadeal"
+	dataDealkeeper "github.com/medibloc/panacea-core/v2/x/datadeal/keeper"
+	dataDealtypes "github.com/medibloc/panacea-core/v2/x/datadeal/types"
```

Comment:
How about changing it to all lowercase?
Although it is an alias, it is a package, and all other aliases are used in lowercase letters.

```suggestion
	datadealkeeper "github.com/medibloc/panacea-core/v2/x/datadeal/keeper"
	datadealtypes "github.com/medibloc/panacea-core/v2/x/datadeal/types"
```

Reply:
I see.
I'll change all aliases with lowercase.

Reply:
Good. Thank you :)